Horizon/Horizon/Sources/Features/Notebook/Common/Domain/CombineMakeRequestExtension.swift

```diff
@@ -0,0 +1,30 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2025-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Combine
+import Core
+
+extension API {
+    func makeRequest<Request: APIRequestable>(_ requestable: Request) -> AnyPublisher<Request.Response?, Error> {
+        let apiResponseSubject = PassthroughSubject<Request.Response?, Error>()
+        makeRequest(requestable) { response, _, _ in
+            apiResponseSubject.send(response)
+        }
+        return apiResponseSubject.eraseToAnyPublisher()
+    }
```

Comment:
This won't forward errors. We need to null check the third `Error?` parameter. Instead of using a subject you can use a Future or even better, use CombineExt's AnyPublisher.create. 
```
AnyPublisher<Request.Response, Error>.create { subscriber in
    // send value or error
    return AnyCancellable {}
}
```